astropy/io/misc/asdf/tags/transform/basic.py

```diff
@@ -13,7 +13,7 @@
 
 
 class TransformType(AstropyAsdfType):
-    version = '1.1.0'
+    version = '1.2.0'
```

Comment:
Currently this change raises an error because transform v 1.2.0 is not in the currently released `version_mapping` file.